doc/userguide/capture-hardware/ebpf-xdp.rst

```diff
@@ -100,6 +99,11 @@ Now, you can build and install the library ::
  sudo make install_headers
  sudo ldconfig
 
+You may miss some i386 headers (as eBPF is not x86_64 and some included headers
```

Comment:
I'm not sure what you're saying here ... `may miss some i386 headers (as eBPF is not x86_64)`. 

Does `may miss` mean `will miss` or `may or may not`? 

Suggest "If your platform is x86_64, install architecture specific headers``

Reply:
I derived this from the original doc. I actually didn't need it but left it there. The original docs also mention `Some i386 headers will also be needed as eBPF is not x86_64 and some included headers are architecture specific` 

With that `may miss` I wanted to say that it might be possible that the compilation will complain about missing i386 headers - I actually didn't need any of that but left it there just to be sure...

Reply:
Would:
"As eBPF is not x86_64, your installation might miss some i386 headers that are architecture-specific."
Work?

Reply:
I honestly think we don't need this here at all unless you can think of some scenario where we might. 
Do you have any opinion on this @victorjulien @jasonish

Reply:
I'm not familiar enough with this to comment.  I do know I just have to install the ebpf-devel packages on RHEL based systems to build with ebpf support, but that might not cover the individual ebpf programs.